src/utils.js

```diff
@@ -0,0 +1,60 @@
+import dayjs from 'dayjs';
+
+// Функция из интернета по генерации случайного числа из диапазона
+// Источник - https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_random
+const getRandomInteger = (a = 0, b = 1) => {
+  const lower = Math.ceil(Math.min(a, b));
+  const upper = Math.floor(Math.max(a, b));
+
+  return Math.floor(lower + Math.random() * (upper - lower + 1));
+};
+
+// Генерирует уникальную коллекцию
+const generateUniqCollection = (collection) => {
+  const mySet = new Set(collection);
+  const uniqArray = Array.from(mySet);
+
+  return uniqArray;
+};
+
+// Достает случайный элемент из массива
+const getRandomElement = (array) => {
+  const randomIndex = getRandomInteger(0, array.length - 1);
+
+  return array[randomIndex];
+};
+
+// Приводит в нужный формат дату и время
+const humanizeDate = (date, format) => dayjs(date).format(format);
+
+// Приводит в нужный формат длительность фильма из минут в часы и минуты
+const humanizeFilmRuntime = (item) => {
+  const hours = Math.floor(item / 60);
+  const minutes = item - (60 * hours);
+  const runtime = `${hours}h ${minutes}m`;
+
+  return runtime;
```

Comment:
Вообще, лучше в таких местах не создавать переменную, а сразу возвращать результат выражения, потому, что переменная здесь не имеет смысла. У тебя это часто повторяется
```suggestion
  return `${hours}h ${minutes}m`;
```